### PR TITLE
Various grammatical fixes and minor clarifications.

### PR DESCRIPTION
**Pull request by tinynumbers**
Fixed some unnecessary commas and minor grammatical issues.  Expanded a few things to further clarify.

### CODE REVIEW

File: source/guides/actions/exception-handling.md [hunk 1]
```diff
@@ -81,12 +81,11 @@ module Web::Controllers::Dashboard
 end
 ```
 
-If we specify a symbol as the value for `handle_exception`, it will be used to manage the exception.
-In our case we want to protect the action from unwanted access: only admins are allowed.
+If we specify the name of a method (as a symbol) as the value for `handle_exception`, this method will be used to resond to the exception.
```
Comment (joneslee85) [inline source/guides/actions/exception-handling.md]: s/resond/respond

File: source/guides/actions/http-caching.md [hunk 2]
```diff
@@ -56,16 +56,16 @@ end
 ## Conditional GET
 
 _Conditional GET_ is a two step workflow to inform browsers that a resource hasn't changed since the last visit.
-At the end of the first request, it receives special HTTP response headers to send back the next time it will come back.
-If that header matches the value that the server calculates, then the resource is still cached and a `304` status (Not Modified) is returned.
+At the end of the first request, the respons includes special HTTP response headers that the browser will use next time it comes back.
```
Comment (joneslee85) [inline source/guides/actions/http-caching.md]: s/respons/responses

File: source/guides/actions/control-flow.md [hunk 3]
```diff
@@ -153,7 +153,7 @@ module Web::Controllers::Dashboard
 end
 ```
 
-`#halt` accepts a HTTP status code as the first argument.
+`#halt` accepts an HTTP status code as the first argument.
```
Comment (joneslee85) [inline source/guides/actions/control-flow.md]: s/an/a

Comment (tinynumbers) [inline source/guides/actions/control-flow.md]: I think that could go either way - I (in American English) pronounce HTML as "aitch-tee-em-el" which indicates that "an" is proper.  See http://english.stackexchange.com/questions/25742/is-it-supposed-to-be-a-html-or-an-html for discussion.

**Pull request merged**